README.md

```diff
@@ -12,26 +12,32 @@
 * [License](#license)
 
 ## Description ✏️
-[Babble](https://babble-lilac.vercel.app/) provides live video chat and text field services. It enables people from around the world to work together. Users can create rooms with passwords, invite people to their rooms, live video chat and type in a shared text field.
+[Babble](https://www.letsbabble.tech/) is a web application that provides live video chat and text field services. It enables people from around the world to work together. Users can create rooms, invite people, live video chat and type in a shared text field. In the future, you will be able to match with a random Babble user, even if they are on the other side of the world. Connect with programmers from all around the world!
 
 ## Demo 💾
-Live demo available here: https://babble-lilac.vercel.app/
+Live demo available here: https://www.letsbabble.tech/
 
 ## Site 🖥️
 
 ### Landing Page
-![image of babble landing page]()
+![image of babble landing page](images/Landing.png)
 
-### Rooms
-![image of babble rooms page]()
+### Selection
+![image of babble selections page](images/Selection.png)
+
+### Join
+![image of babble join meeting page](images/Join.png)
+
+### Video
+![image of babble video meeting page](images/Video.png)
 
 ## Technologies ⚙️
 * [Vue.js](https://vuejs.org/) with [Tailwind CSS](https://tailwindcss.com/)
 * [PeerJS](https://peerjs.com/)
 * [Firepad](https://firepad.io/)
 
 ## Usage 💡
-* Open [Babble](https://babble-lilac.vercel.app/).
+* Open [Babble](https://www.letsbabble.tech/).
 * Create a new room and/or join a room.
 * Chat and type!
```

Comment:
I think we can replace or add onto this with a section for requirements to run the project locally. If we have a contributing file, placing the commands to create the development environment is useful.

Reply:
Yeah, that makes sense. Definitely adding an installation section is a good idea. I think the usage the way it is can stay. Maybe just mention the self hosted url

Reply:
For sure, I just thought that there is a decent amount of environment variables required for Firebase locally as well. Maybe just a note that a Firebase app must be configured with the following credentials in a `.env` file (like this) for any of the code editing features to work locally:

```
VUE_APP_FB_APP_ID=
VUE_APP_FB_DB_URL=
VUE_APP_FB_AUTH_DOMAIN=
VUE_APP_FB_API_KEY=
```

Reply:
Yeah we can add this part under configuration